---
/**:
  ros__parameters:
    update_rate: 10.0
    timeout_driving_capability: 0.5
    data_ready_timeout: 30.0
    use_parking_after_stopped: false
    use_emergency_hold: false
    emergency_hazard_level: 2 # 1: Safe Fault, 2: Latent Fault, 3: Single Point Fault
